python/test/math/Test_scion_math_IntervalDomain.py

```diff
@@ -0,0 +1,62 @@
+# standard imports
+import unittest
+import sys
+
+# third party imports
+
+# local imports
+from scion.math import IntervalDomain
+
+class Test_scion_math_IntervalDomain( unittest.TestCase ) :
+    """Unit test for the IntervalDomain class."""
+
+    def test_component( self ) :
+
+        def verify_chunk( self, chunk ) :
+
+            # verify content
+            self.assertAlmostEqual( -1., chunk.lower_limit )
+            self.assertAlmostEqual(  1., chunk.upper_limit )
+
+            # verify functions
+            max = sys.float_info.max
+            min = -max
+
+            self.assertEqual( False, chunk.is_inside( x = min ) );
+            self.assertEqual( True, chunk.is_inside( x = -1.0 ) );
+            self.assertEqual( True, chunk.is_inside( x = -0.5 ) );
+            self.assertEqual( True, chunk.is_inside( x =  0.0 ) );
+            self.assertEqual( True, chunk.is_inside( x =  0.5 ) );
+            self.assertEqual( True, chunk.is_inside( x =  1.0 ) );
+            self.assertEqual( False, chunk.is_inside( x = max ) );
+
+            self.assertEqual( False, chunk.is_inside( x = min ) );
```

Comment:
is_contained should be used instead

Reply:
Yes, I noticed this last week and fixed it in #42